sources/api/types.go

```diff
@@ -57,19 +57,27 @@ func (a *AggregateData) Merge(b *AggregateData) {
 	a.Events = append(a.Events, b.Events...)
 }
 
+type ContainerSpec struct {
+	cadvisor.ContainerSpec
+	CpuRequest    int64
+	MemoryRequest int64
+}
+
+type ContainerStats struct {
+	cadvisor.ContainerStats
+}
+
 type Container struct {
 	Hostname   string
 	ExternalID string
 	Name       string
 	Image      string
-	// TODO(vishh): Consider defining an internal Spec and Stats API to guard against
```

Comment:
+1. We can get rid of cadvisor's Spec and Stats completely. The model does that.